docker/ds/bootstrap/setup-idm.sh

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
dsconfig has a batch mode.
It would be faster to call dsconfig once with all commands in a batch.

Reply:
Good point. Thanks Ludo.